packages/mobile/components/Breadcrumbs.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { View, Text, TouchableOpacity } from "react-native";
+import styled from "styled-components";
+
+const StyledTouchableOpacity = styled(TouchableOpacity)``;
+
+const StyledView = styled(View)`
+  flex-direction: row;
+  background-color: #fff;
+  min-width: 100%;
+  max-width: 100%;
+  padding-top: 9px;
+  padding-left: 19px;
+  font-size: 12px;
+  line-height: 16px;
+`;
+
+const StyledText = styled(Text)`
+  color: #bdbdbd;
+  letter-spacing: 0.4px;
+`;
+
+const StyledTextSeparator = styled(StyledText)`
+  padding-left: 8px;
+  padding-right: 8px;
+`;
+
+const BreadcrumbItem = ({ title }: { title: string }) => (
+  <StyledText>{title}</StyledText>
+);
+
+const BreadcrumbSeparator = ({ separator }: { separator: string }) => (
+  <StyledTextSeparator>{separator}</StyledTextSeparator>
+);
+
+const BreadcrumbCollapser = () => <StyledText>...</StyledText>;
+
+const Breadcrumbs = ({
```

Comment:
Great job on deconstructuring props!, as a suggestoin I'd say create an interface with the types so it improves readability.

````
interface Props {
    separator: string;
    items: { title: string }[];
}

const Breadcrumbs = ({ separator, items }: Props) => {
.
.
.
````